_zip/missingapi.yml

```diff
@@ -1,72 +1,42 @@
 ### YamlMime:XRefMap
 hrefUpdated: true
 references:
-- uid: System.Xml.Linq.XElement.Elements*
-  href: https://msdn.microsoft.com/library/system.xml.linq.xelement.elements(v=vs.110).aspx
-  name: Elements
-  name.vb: Elements
-  fullName: XElement.Elements
-  fullName.vb: XElement.Elements
-- uid: System.Xml.Linq.XElement.ToString*
-  href: https://msdn.microsoft.com/library/system.xml.linq.xelement.tostring(v=vs.110).aspx
-  name: ToString
-  name.vb: ToString
-  fullName: XElement.ToString
-  fullName.vb: XElement.ToString
-- uid: System.Xml.Linq.XDocument.ToString*
-  href: https://msdn.microsoft.com/library/system.xml.linq.xdocument.tostring(v=vs.110).aspx
-  name: ToString
-  name.vb: ToString
-  fullName: XDocument.ToString
-  fullName.vb: XDocument.ToString
-- uid: System​.Runtime​.CompilerServices.IndexerNameAttribute
+- uid: System.Runtime.CompilerServices.IndexerNameAttribute
```

Comment:
Should this be here? IndexerNameAttribute is present in [API Browser](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.indexernameattribute)